docs/hugo/content/introduction/breaking-changes/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Breaking Changes"
+linkTitle: "Breaking Changes"
+---
+We go to great lengths to avoid breaking changes as much as possible, as we're well aware that they can cause issues for our users. However, they do occasionally occur, so we've committed to providing good documentation each time this occurs.
+
+## Released Breaking Changes
+
+### v2.0.0
+
+Breaking changes are:
+
+* Upgrades from releases prior to v2.0.0-beta.5 are disallowed
+* Alpha CRD versions have been removed
+
```

Comment:
Missing mention on the `ResourceGroup` status change?

Reply:
Added.